project_national/sdr_upgrades_tmy3.yml

```diff
@@ -386,19 +386,6 @@ references:
 ### Standard Data Release - Residential -- ###
 upgrades:
   ### -- HVAC: Air Source Heat Pumps -- ###
-  - upgrade_name: Cold Climate Heat Pump with Electric Backup
-    package_apply_logic: *remove_high_rise_shared
-    options:
-      - *ducted_ashp_seer_20_11_hspf_cchp_electric_backup
-      - *cooling_setpoint_no_offset
-      - *cooling_setpoint_offset_magnitude_0
-      - *cooling_setpoint_offset_period_none
-      - *heating_setpoint_no_offset
-      - *heating_setpoint_offset_magnitude_0
-      - *heating_setpoint_offset_period_none
-      - *heat_pump_cooling_efficiency_ducted
-      - *partial_conditioning_100_percent
```

Comment:
Don't forget to remove test/base_results/upgrades/sdr_annual/results_up18.csv after this change.

Reply:
Thanks, Done: [4fe266f](https://github.com/NREL/resstock/pull/1413/commits/4fe266f5def8afd8df54cad2a0490e2aa8bb4c37)[4fe266f](https://github.com/NREL/resstock/pull/1413/commits/4fe266f5def8afd8df54cad2a0490e2aa8bb4c37)[4fe266f](https://github.com/NREL/resstock/pull/1413/commits/4fe266f5def8afd8df54cad2a0490e2aa8bb4c37)